File: 1482-how-many-numbers-are-smaller-than-the-current-number/how-many-numbers-are-smaller-than-the-current-number.java
class Solution {
    //Rehan I Baig, gitHub:@raaayhaan
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int[] freq = new int[nums.length];
        for(int i = 0; i < nums.length; i++){
            for (int j = 0; j < nums.length; j++){
                if(nums[i] > nums[j]){
                    freq[i]++;
                }
            }
        }
        return freq;
    }
}

File: 1482-how-many-numbers-are-smaller-than-the-current-number/how-many-numbers-are-smaller-than-the-current-number.py
class Solution:
    #Rehan I Baig, gitHub:@raaayhaan
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        freq = [0] * len(nums)
        for i in range(len(nums)):
            for j in range(len(nums)):
                if nums[i] > nums [j]:
                    freq[i] += 1
        return freq
        